kong/plugins/aws-lambda/handler.lua

```diff
@@ -186,9 +186,11 @@ function AWSLambdaHandler:access(conf)
 
   -- TRACING: set KONG_WAITING_TIME stop
   local ctx = ngx.ctx
+  local lambda_wait_time_total = get_now() - kong_wait_time_start
   -- setting the latency here is a bit tricky, but because we are not
   -- actually proxying, it will not be overwritten
-  ctx.KONG_WAITING_TIME = get_now() - kong_wait_time_start
+  ctx.KONG_WAITING_TIME = lambda_wait_time_total
```

Comment:
It seems that AWS_LAMBDA_WAIT_TIME can't be deducted correctly in header_filer if an error happens during the invocation of lambda.

Reply:
Hmm good catch, a potential problem indeed... let me check if I can find a proper way to wrap around `error`

Reply:
I suppose we can put this at Line 177 so as that no matter the invocation successes or not, we can get the waiting_time.

Reply:
Fair point. Fixed